source/administration/changelog.md

```diff
@@ -12,7 +12,7 @@ Also see [changelog in progress](http://bit.ly/2nK3cVf) for the next release.
 ### Highlights
 
 #### Incident Management provided out-of-the-box (E20)
- - Pre-packaged and pre-installed the channel export plugin for enterprise-ready builds.
+ - Prepackaged and pre-installed the Incident Management as well as Channel Export plugins for enterprise-ready builds.
```

Comment:
Sorry @amyblais it was my mistake, `Pre-packaged` should also have an hyphen. Otherwise LGTM!